### DISCUSSION TITLE
Icon Request - Inverse of arrows-expand (arrows-contract?)

### DISCUSSION THREAD

**Discussion by robertrbairdii** [tailwindlabs/heroicons#395] [Ideas]
There's currently the `arrows-expand` icon, it'd be really nice to have the opposite version with the arrows pointing inward.

As an example use case, we're using the `arrows-expand` icon on a button to make our application go full screen, we'd likely use an icon with inward facing arrows on our exit fullscreen mode button.

**Comment by viniciussbs** [comment]
Same here. I'm building a modal window that can be full screen (not native full screen, but full viewport).

![image](https://user-images.githubusercontent.com/245791/91663502-905eb380-eabf-11ea-86e0-5db848d32633.png)

**Comment by ecklf** [comment]
@sschoger quickly made the suggested files for you. I also did notice a mistake that is addressed in #117 and also fixed that for those new icons.

![Screen Shot 2020-10-02 at 10 10 52](https://user-images.githubusercontent.com/8146736/94901879-9f0e0100-0497-11eb-82c5-d450a414eaa5.png)
